live/live.py

```diff
@@ -0,0 +1,76 @@
+# Using live as m3u8 and DLNA media renderer
+#
+# Macast Metadata
+# <macast.title>Live Renderer</macast.title>
+# <macast.renderer>LiveRenderer</macast.renderer>
+# <macast.platform>darwin,win32,linux</macast.platform>
+# <macast.version>0.1</macast.version>
+# <macast.host_version>0.7</macast.host_version>
+# <macast.author>NOTAG</macast.author>
```

Comment:
The author name should be your github id, which is dushan555
Macast will read this value to correctly display your GitHub avatar